src/vm/functions/mod.rs

```diff
@@ -220,10 +220,14 @@ fn native_begin(mut args: Vec<Value>) -> Result<Value> {
     }
 }
 
-fn native_print(input: Value) -> Result<Value> {
+fn special_print(args: &[SymbolicExpression], env: &mut Environment, context: &LocalContext) -> Result<Value> {
```

Comment:
You should add cost tracking to this function -- it should just be a linear function on the size of input.

Reply:
I think it's addressed with https://github.com/blockstack/stacks-blockchain/pull/1319/commits/dc42d045826b42bdae8d11624e1700b804b2c958, but could be missing something.